articles/container-registry/container-registry-faq.md

```diff
@@ -441,3 +441,83 @@ This setting also applies to the `az acr run` command.
 ## Next steps
 
 * [Learn more](container-registry-intro.md) about Azure Container Registry.
+
+# Error references for `az acr check-health`
```

Comment:
@ni-reis - Pls make this an H2 heading (`## Error reference...` and move section **above** the **Next steps** section.